demo/specs/quickstart/README.md

```diff
@@ -13,6 +15,21 @@ nvidia-smi -L
 kubectl get pod -A
 ```
 
+#### Choose the appropriate API version directory
+The example manifests are provided in two directories:
+
+- `v1` — for clusters that support **resource.k8s.io/v1**
+- `v1beta1` — for clusters that support **resource.k8s.io/v1beta1**
+
+Choose the directory that matches your cluster’s supported DRA API version:
+```console
+# for resource.k8s.io/v1beta1
+cd v1
+
+# for resource.k8s.io/v1beta1
+cd v1beta1
+```
```

Comment:
I appreciate that this is the result of a review discussion and I don't suggest changing this anymore.

However, it doesn't go unnoticed how kinda quirky this is! :) 🤷 Shouldn't there be a better way? I don't now, maybe the answer is "no, because...".